bittensor/_neuron/text/core_validator/__init__.py

```diff
@@ -588,6 +599,7 @@ def add_args( cls, parser ):
         parser.add_argument('--nucleus.importance', type=float, help='hyperparameter for the importance loss', default=3)
         parser.add_argument('--nucleus.noise_multiplier', type=float, help='Standard deviation multipler on weights', default=2 )
         parser.add_argument('--nucleus.dendrite_backward', action='store_true', help='Pass backward request to the server side or not', default=False )
+        parser.add_argument('--nucleus.scaling_law_power', type=float, help='Power for modified scaling law, powered down to improve dynamic range, e.g. 3 → 6 nats for 0.5.', default=0.5)
```

Comment:
Question: so to improve the dynamic range, we lower this power factor? Ie.  what happens when I set this factor to 2 instead of 0.5?

Reply:
Yes, lower power factor to improve dynamic range. Increase power factor to amplify the scaling law, so a factor of 2 would square the parameter count estimate and drastic enlarge the incentive differences.